yarn.lock

```diff
@@ -3325,6 +3325,7 @@ material-ui@next:
     prop-types "^15.5.10"
     react-event-listener "^0.5.0"
     react-jss "^7.1.0"
+    react-popper "^0.7.2"
```

Comment:
This was added in here automagically when I called `yarn upgrade`. [It's a dependency of material-ui](https://github.com/callemall/material-ui/blob/v1-beta/package.json#L76).